### DISCUSSION TITLE
잘 밤에 쓸데없는 생각하기.../Git flow, GitHub flow, GitLab flow

### DISCUSSION THREAD

**Discussion by giscus** [ujuc/ujuc.github.io#60] [General]
# 잘 밤에 쓸데없는 생각하기.../Git flow, GitHub flow, GitLab flow

Git flow, GitHub flow, GitLab flow 에대해서 좀 알아보자. 머리아프다.

https://ujuc.github.io/2015/12/16/git-flow-github-flow-gitlab-flow/

**Comment by keymon2** [comment]
감사합니다 눈의 틔었내요 호호잇

**Reply by ujuc** [reply]
감사합니다 😄

**Reply by Kernelzero** [reply]
Best gitX-flow article I've read.

**Comment by AppleCEO** [comment]
잘봤습니다~

**Comment by eogml88** [comment]
brunch[브런치]와 branch[브랜치]는 서로 다른 의미의 단어이고 발음도 다릅니다.

**Reply by ujuc** [reply]
엇 감사합니다. 한번 업뎃을 해놔야겠네요 :)
